### PR TITLE
New version of glossary publisher

### PR DESCRIPTION
**Pull request by mereolog**
This implements a SPIN-free version of glossary publisher due to issues reported in #23 
Note that the generated definitions do not include (yet) nested OWL restrictions.

### CODE REVIEW

**Review by trypuz — APPROVED** [review]
Approved

File: publisher/product/glossary/build.sh [hunk 1]
```diff
@@ -115,390 +31,65 @@ function publishProductGlossaryRemoveWarnings() {
 #
 function publishProductGlossaryContent() {
 
-  logRule "Publishing the content files of the glossary product"
-
-  require ontology_product_tag_root || return $?
-  require glossary_product_tag_root || return $?
-
-  local numberOfProductionLevelOntologyFiles=0
+	logRule "Publishing the content files of the glossary product"
```
Comment (przemekgradzki) [inline publisher/product/glossary/build.sh:34]: the formatting is not consistent with that used elsewhere in the code - compare with [deleted line 118](https://github.com/edmcouncil/ontology-publisher/pull/24/files#diff-4be3c5b9229ba89792c3fa046edbbc35d1c6d8ee76d95206275588eec514f315L118)

File: publisher/lib/fibos_collector.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                            print('Cannot parse', file, 'because', str(exception.args))
+                fibo_file.close()
+
+    fibo_dev.serialize(destination=fibo_dev_file_path)
+    fibo_prod.serialize(destination=fibo_prod_file_path)
+
+
+def __get_fibo_prod_ontology_iri_local_parts(fibo_folder_path: str) -> list:
+    fibo_prod_ontolog_iri_local_parts = []
+    prod_fibo_file = open(os.path.join(fibo_folder_path, 'AboutFIBOProd.rdf'))
```
Comment (przemekgradzki) [inline publisher/lib/fibos_collector.py:60]: there are a lot of hard-coded FIBO-related references

File: publisher/product/glossary/build.sh [hunk 3]
```diff
@@ -115,390 +31,65 @@ function publishProductGlossaryRemoveWarnings() {
...
-  OPTIONAL {?c owlnames:example ?example}
-  OPTIONAL {?c owlnames:explanatoryNote ?explanatoryNote}
+	touch "${glossary_product_tag_root}/glossary.log"	
 
-  OPTIONAL {?c  owlnames:mdDefinition ?GeneratedDefinition}
-}
-GROUP BY ?c ?Term ?Type ?Definition ?GeneratedDefinition ?example ?explanatoryNote ?ReleaseStatus
-ORDER BY ?Term
-__HERE__
+	${PYTHON3} ${SCRIPT_DIR}/lib/csv-to-xlsx.py \
```
Comment (przemekgradzki) [inline publisher/product/glossary/build.sh:83]: remove spaces after "\" characters (also in following lines)

File: publisher/product/glossary/build.sh [hunk 4]
```diff
@@ -115,390 +31,65 @@ function publishProductGlossaryRemoveWarnings() {
...
-  ${SED} -i '1s/\t[?]/,/g;1s/^[?]//' "${glossaryBaseName}-dev.csv"
-  ${SED} -i '1s/\t[?]/,/g;1s/^[?]//' "${glossaryBaseName}-prod.csv"
 
-  touch "${glossary_product_tag_root}/glossary.log"
-  
-  ${PYTHON3} ${SCRIPT_DIR}/lib/csv-to-xlsx.py \
-    "${glossaryBaseName}-prod.csv" \
-    "${glossaryBaseName}-prod.xlsx" \
-    "${glossary_script_dir}/csvconfig"
+	${PYTHON3} ${SCRIPT_DIR}/lib/csv-to-xlsx.py \
```
Comment (przemekgradzki) [inline publisher/product/glossary/build.sh:89]: remove spaces after "\" characters (also in following lines)

**Review by przemekgradzki — CHANGES_REQUESTED** [review]

File: publisher/lib/fibos_collector.py (hunk 2 above)
Comment (mereolog) [reply publisher/lib/fibos_collector.py:60]: Is there a pattern I can follow?
I looked into a couple of shell files and found multiple hard-coded FIBO-related references all over the place, e.g., in https://github.com/edmcouncil/ontology-publisher/blob/master/publisher/product/ontology/build.sh.
Similarly for the python scripts.

File: publisher/product/glossary/build.sh (hunk 1 above)
Comment (mereolog) [reply publisher/product/glossary/build.sh]: This should be fixed by the patch.

File: publisher/product/glossary/build.sh (hunk 3 above)
Comment (mereolog) [reply publisher/product/glossary/build.sh]: This should be fixed by the patch.

File: publisher/product/glossary/build.sh (hunk 4 above)
Comment (mereolog) [reply publisher/product/glossary/build.sh]: This should be fixed by the patch.

File: publisher/lib/fibos_collector.py (hunk 2 above)
Comment (mereolog) [reply publisher/lib/fibos_collector.py]: I managed to generalised FIBO references up to the level of Docker, i.e., python script and shell script are FIBO-free or at least are less FIBO-dependent.
I am not able to generalise them further.

**Review by przemekgradzki — APPROVED** [review]

**Pull request merged**